MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,781 @@
+---
+layout: post
+title: Working with DataForm in .NET MAUI DataForm control | Syncfusion
+description: Learn about the working with DataForm in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Working with DataForm in Maui DataForm (SfDataForm)
+
+## Auto generate data editor 
+
+By default, the `SfDataForm.Items` will be generated based on the property type. For example, the `DataFormTextItem` will be generated based on the property type in `DataObject`. 
+The `DataFormItem` generation depends on the type and attribute defined for the property.
+The following tables lists the several types of `DataFormItem` and its constraints for auto generation:
```

Comment:
You can customize the auto generated editor, label and group settings by using the `GenerateDataFormItem` event.

Reply:
added